pages/interface/components/Header/index.js

```diff
@@ -77,6 +78,17 @@ export default function HeaderComponent() {
 
       {user && (
         <>
+          <PrimerHeader.Item
+            sx={{
+              mr: 2,
```

Comment:
```suggestion
            sx={{
              display: ['none', 'flex'],
              mr: 2,
```

Conforme conversas que eu mencionei, é melhor não adicionar esse botão na versão mobile. E não vai ser legal adicionar e depois remover, então é melhor já não adicionar.